src/nlp/arrow_dataset.py

```diff
@@ -1419,8 +1482,8 @@ def train_test_split(
         generator: Optional[np.random.Generator] = None,
         keep_in_memory: bool = False,
         load_from_cache_file: bool = True,
-        train_cache_file_name: Optional[str] = None,
-        test_cache_file_name: Optional[str] = None,
+        train_indices_cache_file_name: Optional[str] = None,
```

Comment:
This is a bit long but I think it's important that the user does not mistake this cache for the dataset table cache.